source/chapters/user_interface.rst

```diff
@@ -1168,3 +1135,36 @@ The features in detail:
 
 .. seealso:: For more information, go to the chapter
              :ref:`library-previewing-tracks`.
+
+.. _interface-expansion-buttons:
+
+Section Expansion Buttons
```

Comment:
I suggest to make rename this to **Skin Settings** and move it to the top so it's the first chapter in **3. An Overview of the Mixxx Interface**.
Yo don't need to do that in this PR, but if you do please incorporate the rephrasing suggestions below.
@Holzhaus Objections?

Reply:
Nope.

Reply:
Done!